xbmc/addons/AddonDatabase.cpp

```diff
@@ -900,11 +930,12 @@ bool CAddonDatabase::GetDisabled(std::set<std::string>& addons)
     if (!m_pDS)
       return false;
 
-    std::string sql = PrepareSQL("SELECT addonID FROM installed WHERE enabled=0");
+    std::string sql = PrepareSQL("SELECT addonID, disabledReason FROM installed WHERE enabled=0");
```

Comment:
How is there for case that all disabled addons want to get?

Reply:
This will get them all. It’s just it gets the reason for them too.

Reply:
Sorry, bad moment in my mind (was something there in my head that it select adonid with matching reason)